bika/lims/browser/client/views/analysisspecs.py

```diff
@@ -5,77 +5,21 @@
 # Copyright 2018 by it's authors.
 # Some rights reserved. See LICENSE.rst, CONTRIBUTORS.rst.
 
+from bika.lims import api
 from bika.lims import bikaMessageFactory as _
-from bika.lims.browser import BrowserView
-from bika.lims.browser.bika_listing import BikaListingView
-from bika.lims.permissions import *
-from bika.lims.utils import isActive
-from bika.lims.utils import tmpID
-from plone.app.layout.globals.interfaces import IViewView
-from Products.CMFCore.utils import getToolByName
-from Products.CMFPlone.utils import _createObjectByType
-from zope.interface import implements
+from bika.lims.controlpanel.bika_analysisspecs import BaseAnalysisSpecsView
+from bika.lims.permissions import AddAnalysisSpec
 
 
-class ClientAnalysisSpecsView(BikaListingView):
-    implements(IViewView)
+class ClientAnalysisSpecsView(BaseAnalysisSpecsView):
```

Comment:
Better inherit from the `BikaListingView`, because this base class is only indirection.